warehouse/tasks.py

```diff
@@ -90,6 +95,8 @@ def get_request(self):
             registry = self.app.pyramid_config.registry
             env = pyramid.scripting.prepare(registry=registry)
             env["request"].tm = transaction.TransactionManager(explicit=True)
+            env["request"].timings = {"new_request_start": time_ms()}
```

Comment:
```suggestion
            env["request"].timings = {"new_request_start": time.time()*1000}
```

Reply:
(just realized I didn't write unit tests for these 2 lines so I'm adding too. we never know)